imports/plugins/custom/payments-paystack/client/checkout/paystack.js

```diff
@@ -0,0 +1,151 @@
+/* eslint camelcase: 0 */
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { AutoForm } from "meteor/aldeed:autoform";
+import { $ } from "meteor/jquery";
+import { Reaction } from "/client/api";
+import { Cart, Packages } from "/lib/collections";
+import { Paystack } from "../../lib/api";
+import * as Collections from "/lib/collections";
+import { PaystackPayment } from "../../lib/collections/schemas";
+
+import "./paystack.html";
+
+const submitting = false;
+
+function uiEnd(template, buttonText) {
+  template.$(":input").removeAttr("disabled");
+  template.$("#btn-complete-order").text(buttonText);
+  return template.$("#btn-processing").addClass("hidden");
+}
+
+function paymentAlert(template, errorMessage) {
+  // return $(".alert").removeClass("hidden").text(errorMessage);
```

Comment:
Why this comment too?! And it almost seems like nothing would be returned

Reply:
will remove the comment, it doesn't have to return aything to work tho